verif/big_core/tests/primary_numbers.c

```diff
@@ -1,4 +1,4 @@
-#include <stdio.h>
+#include "../../../app/defines/big_core_defines.h"
```

Comment:
Please add a header to each C file.
(You are welcome to just use chatGPT to write this information for you)

The header should have name/owner, date,  etc.
something like this:

//====================================================
// Author        : John Doe
// Date          : 01/16/2023
// Project       : MyProject
// Purpose       : A simple program that demonstrates the basic usage of the printf function from the standard library.
// Description   : This program prints "Hello World!" to the console as its output. 
//                        
//
// References    : N/A
// Notes         : N/A
//====================================================

Reply:
Done